Detector/DetCEPCv4/src/tracker/VXD04_geo.cpp

```diff
@@ -791,14 +791,14 @@ static Ref_t create_element(Detector& theDetector, xml_h e, SensitiveDetector se
 								Z))  );
 
 	  if(active_side_band_electronics_option==1)
-	    pv_el_band_pos.addPhysVolID("layer", LayerId ).addPhysVolID( "module" , int(elec_loop)  ).addPhysVolID("sensor", 1 ).addPhysVolID("side", 1 )   ;
+	    pv_el_band_pos.addPhysVolID("layer", LayerId ).addPhysVolID( "module" , int(elec_loop)  ).addPhysVolID("sensor", 1 ).addPhysVolID("side", 0 ).addPhysVolID("barrelside", 1 )   ;
```

Comment:
Hi @fucd ,
Could write more information in the description of your PR? It would be necessary to let people know why you add the field `barrelside`.

Tao

Reply:
Hi @mirguest 

Yes, it is helpful. Thanks your comments. I have update the description.

Chengdong